docs/tutorials/distributed/distributed_slowfast.py

```diff
@@ -42,66 +42,59 @@
 # How to use our code to train a SlowFast model in a distributed manner?
 # ----------------------------------------------------------------------
 #
-# In order to perform distributed training, you need to install MXNet and prepare the cluster ready.
-# MXNet provides a script ``tools/launch.py`` to make it easy to launch distributed training on a cluster with ssh, mpi, sge or yarn.
+# In order to perform distributed training, you need to (1) prepare the cluster ready; (2) install MXNet; and (3) prepare your code
+# and data ready.
 
 ################################################################
-# First, let's install MXNet.
+# We need a cluster that each node can communicate with each other.
+# The first step is to generate ssh keys for each machine.
+# For better illustration, let's assume we have four machines, node1, node2, node3 and node4.
+# We use each of the machine as both server and worker.
+# Note that, we can also have dedicated CPU machines as servers.
+#
+# First, ssh into node1 and type
 # ::
 #
-#     pip install mxnet-cu100
+#     ssh-keygen -t rsa
 #
-# For more installation options (i.e., different versions of MXNet or CUDA),
-# please check `GluonCV installation guide <https://gluon-cv.mxnet.io/install/install-more.html>`_ for more information.
+# Just follow the default, you will have a file named ``id_rsa`` and a file named ``id_rsa.pub``, both under the ``~/.ssh/`` folder.
+# ``id_rsa`` is the private RSA key, and ``id_rsa.pub`` is its public key.
 
+################################################################
+# Second, copy both files (``id_rsa`` and ``id_rsa.pub``) of node1 to all other machines.
+# For each machine, you will fine an ``authorized_keys`` file under ``~/.ssh/`` folder as well.
```

Comment:
`fine` -> `find`. 
It exists on ec2 but might be missing sometimes. Shall we also add 
Q2: I didn't find the file `.ssh/authorized_keys`
Answer: just create one with 
```
touch ~/.ssh/authorized_keys
chmod 600 ~/.ssh/authorized_keys
```